provisioning/etcd/generate.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+cd "$(dirname "$0")"
+helm template templates-api-etcd bitnami/etcd -f ./values.yaml > etcd.yaml
+
+# Replace root password by ENV
+sed -i 's/etcd-root-password: ".*"/etcd-root-password: "$ETCD_ROOT_PASSWORD_BASE64"/g' etcd.yaml
+
+# Replace replica count by ENV
+sed -i 's/replicas: .*/replicas: $ETCD_REPLICA_COUNT/g' etcd.yaml
+sed -i 's/value: "templates-api-etcd-0=.*"/value: "$ETCD_INITIAL_CLUSTER"/g' etcd.yaml
+
+# Modify labels generated by Helm
+sed -i 's/app.kubernetes.io\/instance: templates-api-etcd/app: templates-api-etcd/g' etcd.yaml
+sed -i '/app.kubernetes.io\/name/d' etcd.yaml
+sed -i '/app.kubernetes.io\/managed-by/d' etcd.yaml
+sed -i '/helm.sh\/chart/d' etcd.yaml
```

Comment:
Ještě tady něco asi bude špatně, ale takhle na první pohled nic nevidím. 🤔 
![CleanShot 2022-08-23 at 09 37 28](https://user-images.githubusercontent.com/215660/186099851-413ff3e8-c53c-4ac8-a0be-31753461f147.jpg)

Reply:
Tak na řádku 8: 
![CleanShot 2022-08-23 at 09 41 50](https://user-images.githubusercontent.com/215660/186100666-540badfc-216f-4325-afde-1db41d87e89e.jpg)
Já `sed` moc nepoužívám, tak budu muset otevřít manuál. 😃

Reply:
Pravdepodobne budes mat asi nejaku inu verziu, ... skusim to vsetko hodit do nasho `dev` image, nech to je rovnake.

Reply:
Jo říkám si, že by to mohlo být mac-specific. 🙂

Reply:
Pridane, skus prosim:
```
docker-compose build
docker-compose run --rm -u "$UID:$GID" --service-ports dev bash
./provisioning/etcd/generate.sh
```

Reply:
https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux

Reply:
Dik, uz by to malo ist aj nativne na MacOs: https://github.com/keboola/keboola-as-code/pull/770/commits/f5f165c55a383ff3f3c4d4c67371023500b34219

Reply:
Teda, malo by ti uz aj pregenerovanie cez Helm, ale klucove skontrolovat tuto vygenerovanu definiciu:
https://github.com/keboola/keboola-as-code/blob/eba3c2ccaee2ba8fa7aceb4da9a868e9fccdb226/provisioning/kubernetes/templates/etcd.yaml